webpack/assets/javascripts/react_app/components/PF4/TableIndexPage/TableIndexPage.js

```diff
@@ -173,7 +173,11 @@ const TableIndexPage = ({
     hasHelpPage && {
       title: __('Documentation'),
       icon: <QuestionCircleIcon />,
-      action: { href: customHelpURL || helpURL() },
+      action: {
+        href: customHelpURL || helpURL(),
+        target: '_blank',
+        rel: 'noopener noreferrer',
```

Comment:
According to [this answer ](https://stackoverflow.com/a/67839804)on StackOverflow, all browsers handle this automatically.

Reply:
Good to know, removed!